controllers/constants/constants.go

```diff
@@ -31,6 +31,9 @@ const (
 	LabelEnableRoute   = "enable-route"
 
 	CapabilityServiceMeshAuthorization = "CapabilityServiceMeshAuthorization"
+
+	ModelMeshServiceAccountName = "modelmesh-serving-sa"
+	KserveServiceAccountName    = "kserve-sa"
```

Comment:
Since this is going to be for oauth-proxy uses, maybe?:
```suggestion
	KserveServiceAccountName    = "kserve-oauth-sa"
```